### PR TITLE
Remove "See also" from README, add rebuild-detector to man examples

### PR DESCRIPTION
**Pull request by maximbaz**
Something like this @AladW? 🙂 

I've tried to guess the syntax of man files but I'm not actually sure how to check if it's good...?

### CODE REVIEW

**Comment by AladW** [comment]
Thanks. You can check man pages by running `man -l foo.1` with `foo.1` the file you want to test.

File: man1/aur.1 [hunk 1]
```diff
@@ -421,6 +421,15 @@ variable towards it:
     $ export PACMAN=/usr/local/bin/mypacman
 .EE
 .
+.SS Rebuilding packages against updated dependencies
```
Comment (AladW) [inline man1/aur.1]: I might leave this section in `aur-build(1)`, if we want to keep it in `aur(1)` I'd write a specific command-line in the `EXAMPLES` section

**Comment by Earnestly** [comment]
Or `man ./foo.1`

File: man1/aur.1 (hunk 1 above)
Comment (maximbaz) [reply man1/aur.1]: Moved to the `NOTES` section of `aur-build(1)`, or you have a better location?

File: man1/aur-build.1 [hunk 2]
```diff
@@ -373,6 +373,13 @@ See
 .BR gpg (1)
 for details.
 .
+.SS Rebuilding packages against updated dependencies
+It is sometimes needed to rebuild packages when their dependencies are
+updated, for example in case of dynamic library linking.
+.PP
+You can use https://github.com/maximbaz/rebuild-detector
```
Comment (AladW) [inline man1/aur-build.1]: Groff has some special syntax for URIs too:
```
.UR https://\:github.com/\:maximbaz/\:rebuild-detector
.UE .
```

File: man1/aur.1 (hunk 1 above)
Comment (AladW) [reply man1/aur.1]: The location looks OK to me

File: man1/aur-build.1 (hunk 2 above)
Comment (maximbaz) [reply man1/aur-build.1]: Nice - seems like even named URLs work, I saw an example in `aur-pkglist`, updated 👍

**Comment by AladW** [comment]
👍

**Pull request merged**